<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Syn_UpdateADCOffsetsDlg</class>
 <widget class="QDialog" name="Syn_UpdateADCOffsetsDlg">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Update ADC Offsets</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QGroupBox" name="VoltagesGroupBox">
     <property name="title">
      <string>Voltages(mV)</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="VddLabel">
        <property name="text">
         <string>Vdd:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="VioLabel">
        <property name="text">
         <string>Vio:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="VledLabel">
        <property name="text">
         <string>Vled:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="VddhLabel">
        <property name="text">
         <string>Vddh:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="VddLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="VioLineEdit"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="VledLineEdit"/>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="VddhLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>142</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="OKPushButton">
     <property name="text">
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="CancelPushButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
